downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
+
+.Prerequisites
+
+* A pre-configured Azure VWAN that is already tested and working.
```

Comment:
Does "tested" not imply the VWAN is already working? Is there something more definite that can indicate this state, like the VWAN has a particular status?